tests/resources/Util.robot

```diff
@@ -83,6 +83,10 @@ Install VIC Appliance To Test Server
     Run Keyword And Ignore Error  Cleanup Datastore On Test Server
     Run Keyword And Ignore Error  Cleanup Dangling Networks On Test Server
     Set Test VCH Name
+    # Set a unique bridge network for each VCH that has a random VLAN ID
+    ${vlan}=  Evaluate  str(random.randint(1, 4093))  modules=random
```

Comment:
As long as you have a unique name, vc should give you a unique vlan id, so I don't see a need for this.